src/types-and-traits.rst

```diff
@@ -2637,6 +2637,28 @@ A :t:`higher-ranked trait bound` is a :t:`bound` that specifies an infinite
 list of :t:`[bound]s` for all possible :t:`[lifetime]s` specified by the
 :s:`ForGenericParameterList`.
 
+:dp:`fls_J9DEsd06Ttu9`
+An :t:`implied bound` is a :t:`bound` that is not expressed in syntax, but is
+is the byproduct of relations between the :t:`[generic parameter]s` of
+:t:`[abstract data type]s` or :t:`[function]s`.
+
+:dp:`fls_IfHRxSasGAih`
+A :t:`reference` of the form ``&'a generic_parameter``, where ``'a`` is a
+:t:`lifetime parameter` and ``generic_parameter`` is an arbitrary
+:t:`generic parameter`, yields :t:`implied bound``
+``generic_parameter: 'a``.
```

Comment:
The generic parameter here can be any type which can be relevant if its `Foo<T>` where `T` is a generic parameter, as the lifetime of `Foo` then depends on the generic parameter `T`. that is `&'a Foo<T>` still implies `Foo<T>: 'a`

Reply:
Fixed.